charts/kubernetes-agent/values.yaml

```diff
@@ -1,11 +1,11 @@
-# Default values for kubernetes-tentacle.
+# Default values for kubernetes-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
 
 image:
-  repository: docker.packages.octopushq.com/octopusdeploy/kubernetes-tentacle
+  repository: docker.packages.octopushq.com/octopusdeploy/kubernetes-agent
```

Comment:
this won't change as the tentacle image will still be `tentacle`

Reply:
Damn, good catch!